VikingQuiz.Api/Controllers/UsersController.cs

```diff
@@ -37,11 +37,11 @@ public IActionResult GetAll()
         }
 
         [HttpGet("{id}")]
-        [Authorize]
+       // [Authorize]
         public IActionResult Get(int id)
         {
-            int userId = User.Claims.GetUserId();
-            User user = userRepository.GetUserById(userId);
+         //   int userId = User.Claims.GetUserId();
+            User user = userRepository.GetUserById(id);
```

Comment:
Remove commented lines. Or add a small explanation to why they need to be temporarily kept.